clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/ops/DeployAppengineAtomicOperation.groovy

```diff
@@ -171,6 +206,16 @@ class DeployAppengineAtomicOperation implements AtomicOperation<DeploymentResult
     return versionName
   }
 
+  static List<String> updateConfigRuntime(List<String> configFiles) {
+    if (!configFiles) {
+      return []
+    } else {
+      return configFiles.collect { configFile ->
+        return configFile.replaceAll(/runtime\s*\:\s*[a-zA-Z]+/, "runtime: custom")
```

Comment:
Seems pretty unlikely, but an app.yaml can have arbitrary user-supplied values (e.g. env variables), so it's possible you could accidentally overwrite something here.

Reply:
OK, i've attempted to make the runtime regex a little more strict, and will only replace the first instance of the string.

Is there a more thorough approach or is this ok for now?

Reply:
Removed app.yaml update code completely.  We can message and enforce better in deck for now.